Update PR Template to Include New Description/Overview Guidelines (#11)
customink/.github: .github/pull_request_template.md

### Stakeholder Overview

<!--

> This section should provide a brief overview of the changes and the motivation for the changes in a 
> manner friendly to non-technical people and to technical people who don't know the context or are not
> subject matter experts on the system or update being made. Consider the logical need for the change, 
> as well as the logical effect of the change, particularly as any end-users would experience it.

Reviewer: I was gonna suggest some minor edits here but then I InkGPT-ed it.

```suggestion
Provide a concise summary of the motivation and the driving force behind them. These explanations are designed to be understandable to individuals without technical knowledge, and to people who may not be familiar with the context or specialize in the particular system or update being discussed. Think about the logical rationale for the modification and its practical impact, especially from the viewpoint of any end-users.
```

Or here is another one

```suggestion
Provide a clear and concise summary of what changes are being made and the reasons for these adjustments. The explanation should be easy to understand for both non-tech savvy individuals and technical personnel who may not be well-acquainted with the context of the changes or may not be experts in the system being updated. Give thought to the logical necessity for the changes, as well as the likely consequences these changes could bring about, especially from the perspective of potential end-users.
```

Good example:
IHP is in the process of updating the way carts get built. This is the core cart creation 
logic that sorts and groups orders into carts. With the upcoming change for the UI, users will 
be able to skip the Press Staging Queue and create a cart based on the flash count for the 
printer they are selecting for.

Good example:
We are currently sending all pageViewTags as custom events to fullstory which results in Fullstory 
throttling Custom Events for our accounts. This prevents us from using custom events for more legitimate 

> users (e.g. custom event to track when a user is in test group)

Reviewer 2: ```suggestion
users (e.g. custom event to track when a user is in test group).
```
Add missing period.

> This change is going to help Swag Management to identify sessions of users in Swag Management test group.

Reviewer 2: ```suggestion
This change is going to help Swag Management to identify sessions of users in the Swag Management test group.
```
